spec/classes/datadog_agent_spec.rb

```diff
@@ -723,9 +723,9 @@
                   let(:params) {{ :service_discovery_backend  => 'docker',
                                   :sd_config_backend          => 'etcd',
                                   :sd_backend_host            => 'localhost',
-                                  :sd_backend_port            => '8080',
+                                  :sd_backend_port            => 8080,
                                   :sd_jmx_enable              =>  true,
-                                  :agent5_enable => true,
+                                  :agent_major_version => 5,
```

Comment:
nit: alignment issue here - it was already there but would be nice to fix. You probably scripted this ;)

Reply:
TBH, we're not really consistent with hash alignment.... Not sure why our linters aren't saying something 🤔